qiskit/visualization/__init__.py

```diff
@@ -17,20 +17,160 @@
 
 .. currentmodule:: qiskit.visualization
 
+
+Overview
+========
+
+The visualization module contain functions that visualizes measurement outcome counts, quantum
+states, circuits, pulses, devices and more.
+
+Prerequisites
+=============
```

Comment:
I'd probably just get rid of the "Overview" and "Prerequisites" section headings, and let their text just be a paragraph that's loose under the page title.  The extra headings don't add much extra information, and add visual clutter.

Reply:
I was thinking about using this as an example for other module api and therefore I tried to enforce a structure. But I agree adding headings (for this specific example at least) is not helpful. Removed the headings.